omnibus/config/projects/updater.rb

```diff
@@ -114,6 +114,7 @@
   extra_package_file "#{systemd_directory}/stop-experiment.path"
   extra_package_file '/etc/datadog-agent/'
   extra_package_file '/var/log/datadog/'
+  extra_package_file '/var/run/datadog-packages'
```

Comment:
This is later referred to as a directory, shouldn't the path end with a trailing `/` ?
I'm worried this will create a file, which would prevent a lock file to be placed inside it